code/popup.html

```diff
@@ -11,80 +11,272 @@
   <link rel="shortcut icon" type="image/png" href="/icon/icon.png"/>
   <title>OpenGenus: World's first Offline Search Engine</title>
   <meta charset='utf-8'>
+   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
You cannot link to online resources. You have to save them locally and link it. Also, place the css code to its own file and link it.

Same applies to the `javascript` code.